src/main/java/io/github/queerbric/queerbrics/datatypes/item/QueerBrickItem.java

```diff
@@ -13,8 +13,15 @@
 import net.minecraft.world.World;
 
 public class QueerBrickItem extends SnowballItem {
+	public String title;
```

Comment:
personally i'm in favour of private fields with a getter -- but this should at least be made final